target_chains/solana/programs/pyth-solana-receiver/Cargo.toml

```diff
@@ -13,7 +13,7 @@ no-entrypoint = []
 no-idl = []
 no-log-ix-name = []
 cpi = ["no-entrypoint"]
-default = []
+default = ["mainnet"]
```

Comment:
This makes it more convenient to test by not having to pass --feature mainnet everytime